app/com/linkedin/drelephant/exceptions/core/ExceptionFingerprintingSpark.java

```diff
@@ -150,29 +182,16 @@ private void processDriverLogs(List<ExceptionInfo> exceptions) {
     try {
       String urlToQuery = buildURLtoQuery();
       long startTimeForFirstQuery = System.nanoTime();
-      String completeURLToQuery = completeURLToQuery(urlToQuery);
+      String completeURLToQuery = procesForDriverLogURL(urlToQuery);
+      logSourceInfo.put(ExceptionInfo.ExceptionSource.DRIVER.name(), urlToQuery + STDERR_URL_CONSTANT + startIndex);
       long endTimeForFirstQuery = System.nanoTime();
```

Comment:
Why using System.nanoTime instead of System.currentTimeMillis as since we are reporting the task completion duration in seconds? Since System.nanoTime is slow, not thread-safe and the duration is in order of seconds, so using System.currentTimeMillis would be efficient.

Reply:
System.nanTime is more accurate , thats why used it but I think its not required in this case hence putting currentTimeMillis